apps/condo/domains/common/components/ChangeHistory/ChangeHistoryDiff.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react'
+
+export const ChangeHistoryDiff: React.FC = (props) => <p {...props} />
```

Comment:
Anyway what the reason to move this part of code to the component? Seems to me it couldn't be used anywhere else